src/renderers/shaders/ShaderChunk/fog_fragment.glsl.js

```diff
@@ -1,6 +1,8 @@
 export default /* glsl */`
 #ifdef USE_FOG
 
+	float fogDepth = precisionSafeLength(fogPosition);
```

Comment:
whitespace
```glsl
float fogDepth = precisionSafeLength( fogPosition );
```

Also, can you squash these commits for clarity?

Reply:
Sorted